go/common/crypto/signature/signer_test.go

```diff
@@ -85,3 +85,18 @@ func TestContext(t *testing.T) {
 	_, err = PrepareSignerMessage(unregCtx, []byte("message"))
 	require.NoError(err, "PrepareSignerMessage should work with unregisered context (bypassed)")
 }
+
+func TestBlacklist(t *testing.T) {
```

Comment:
Testing whether you cannot submit transactions (and that transaction is not accepted by other nodes) with blacklisted key would be useful.

Reply:
You could do that by using `show_tx` (as long as the blacklist is part of the genesis document), you only would need to load the blacklist there as well (similar to how we set the chain domain separation context from the genesis doc).